src/NexusMods.Settings/SettingsManager.cs

```diff
@@ -105,7 +105,8 @@ public IObservable<T> GetChanges<T>() where T : class, ISettings, new()
     {
         return _subject
             .Where(tuple => tuple.Item1 == typeof(T))
-            .Select(tuple => (tuple.Item2 as T)!);
+            .Select(tuple => (tuple.Item2 as T)!)
+            .StartWith(Get<T>());
```

Comment:
I didn't add `StartWith` on purpose. If you check the usages, you'll see that consumers do a normal `Get` before they subscribe to `GetChanges`.

If you really want the observable stream to always contain at least one value, add a parameter to `GetChanges` to configure this behavior. Also don't use `StartWith`, that's for a sequence of elements. Use `Prepend` instead.